src/panel_material_ui/widgets/indicators.py

```diff
@@ -79,8 +79,43 @@ def _update_value(self, *_, **__):
         if self.value == -1 and self.variant == "determinate":
             self.variant = "indeterminate"
 
+# Generalize or move to dashboard example
+# Could not get icon from ReactComponent so included it here
+
+class ChangeIndicator(MaterialWidget):
```

Comment:
I have not documented this one yet. I could not get it working without being a MaterialWidget, so I included it here. But I would like to give it a 2nd iteration later to find a more general API for a such a Card/ Indicator.

Reply:
What's wrong with it being a MaterialWidget?

Reply:
For me it ok. But there could be other Card like indicator widgets. I'm not sure what the most important/ generic are?